sources/packages/api/src/services/application/application.service.ts

```diff
@@ -950,4 +955,52 @@ export class ApplicationService extends RecordDataModelService<Application> {
       ) <= COE_WINDOW
     );
   }
+  /**
+   * Deny the Confirmation Of Enrollment(COE) for an Application.
+   * Updates only applications that have the COE status as required.
+   * @param applicationId application id to be updated.
+   * @param locationId location that is setting the offering.
```

Comment:
It seems that "location that is setting the offering" does not apply to this method.